modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc

```diff
@@ -114,16 +114,16 @@ def ray_fn():
            head_cpus="500m",
            min_memory=1,
            max_memory=1,
-           num_gpus=0,
-           image="quay.io/rhoai/ray:2.23.0-py39-cu121", <5>
-           local_queue="local_queue_name", <6>
+           worker_extended_resource_requests={“nvidia.com/gpu”: 0}, <5>
```

Comment:
This can just be left empty if no GPUs are required, in which case it may just be okay to have this PR just remove the `num_gpus=0` line

Reply:
I think it's good to show an example, and the explanation about the deprecated parameter is in the associated callout. Have a look at the latest updates and let me know what you think?